src/main/java/sirius/biz/tenants/TenantController.java

```diff
@@ -127,15 +133,18 @@ public void tenant(WebContext ctx, String tenantId) {
 
         SaveHelper saveHelper = prepareSave(ctx).withAfterCreateURI("/tenant/${id}").withAfterSaveURI("/tenants");
         saveHelper.withPreSaveHandler(isNew -> {
-            tenant.getTenantData().getPermissions().getPermissions().clear();
-            for (String permission : ctx.getParameters("permissions")) {
-                // Ensure that only real permissions end up in the permissions list,
-                // as roles, permissions and flags later end up in the same vector
-                // therefore we don't want nothing else but tenant permissions in this list
-                if (getPermissions().contains(permission)) {
-                    tenant.getTenantData().getPermissions().getPermissions().add(permission);
+            tenant.getTenantData().getPackageData().getUpgrades().clear();
```

Comment:
shouldn't we add a loadFromContext to PackageData - as there will be more place duplicating this code...